source/Nevermore/Advanced/QueryBuilder.cs

```diff
@@ -273,10 +275,16 @@ public async Task<int> CountAsync(CancellationToken cancellationToken = default)
         {
             var clonedSelectBuilder = selectBuilder.Clone();
             clonedSelectBuilder.AddColumnSelection(new SelectCountSource());
+            clonedSelectBuilder.AddOptions(optionClauses);
             var count = await readQueryExecutor.ExecuteScalarAsync<int>(clonedSelectBuilder.GenerateSelect().GenerateSql(), paramValues, RetriableOperation.Select, commandTimeout, cancellationToken);
             return count;
         }
 
+        IOption GetOption()
+        {
+            return new Option(optionClauses.Select(o => new OptionClause(o)).ToList());
+        }
```

Comment:
Can this method be better named?

Reply:
Oops, I just noticed I don't need that method anymore so I will just delete it.